certificate-policy.md

```diff
@@ -1149,73 +1138,81 @@ DSA: Although FIPS 800-57 says that domain parameters may be made available at s
+
+- A single person shall not be permitted to activate or access any cryptographic module that contains the complete CA private signing key. 
+- CA signature keys may be backed up only under at least two-person control. 
+- Access to CA signing keys backed up for disaster recovery shall be under at least two-person control. 
+- The names of the parties used for two-person control shall be maintained on a list that shall be made available for inspection during by Qualified Auditors.
 
 ### 6.2.3 Private key escrow
+For CA Key Pairs that are either (i) used as Root CA Key Pairs or (ii) Key Pairs generated for a subordinate CA that is not the operator of the Root CA or an Affiliate of the Root CA, or (iii) Key Pairs for the operator of the Root CA or an Affiliate of the Root CA:
+- The CA private keys SHALL never be escrowed
 
 ### 6.2.4 Private key backup
+For CA Key Pairs that are either (i) used as Root CA Key Pairs or (ii) Key Pairs generated for a subordinate CA that is not the operator of the Root CA or an Affiliate of the Root CA, or (iii) Key Pairs for the operator of the Root CA or an Affiliate of the Root CA:
+- The CA private signature keys SHALL be backed up under the same multiperson control as the original signature key. 
+- At least one copy of the private signature key shall be stored off-site. 
+- All copies of the CA private signature key shall be accounted for and protected in the same manner as the original. 
+- Backup procedures shall be included in the CA’s CPS
+
 See Section 5.2.2.
 
 ### 6.2.5 Private key archival
-Parties other than the Subordinate CA SHALL NOT archive the Subordinate CA Private Keys without authorization by the Subordinate CA.
+Parties other than the Subordinate CA SHALL NOT archive the Subordinate CA Private Keys.
 
 ### 6.2.6 Private key transfer into or from a cryptographic module
 If the Issuing CA generated the Private Key on behalf of the Subordinate CA, then the Issuing CA SHALL encrypt the Private Key for transport to the Subordinate CA. If the Issuing CA becomes aware that a Subordinate CA's Private Key has been communicated to an unauthorized person or an organization not affiliated with the Subordinate CA, then the Issuing CA SHALL revoke all certificates that include the Public Key corresponding to the communicated Private Key.
 
 ### 6.2.7 Private key storage on cryptographic module
-The CA SHALL protect its Private Key in a system or device that has been validated as meeting at least FIPS 140 level 3 or an appropriate Common Criteria Protection Profile or Security Target, EAL 4 (or higher), which includes requirements to protect the Private Key and other assets against known threats.
+All CAs SHALL protect its Private Key in a system or device that has been validated as meeting at least FIPS 140 level 3 or an appropriate Common Criteria Protection Profile or Security Target, EAL 4 (or higher), which includes requirements to protect the Private Key and other assets against known threats.
```

Comment:
So for 6.2.6, first sentence needs to change to "what?"  Opening issue #44 to discuss

> If the Issuing CA generated the Private Key on behalf of the Subordinate CA, then the Issuing CA SHALL encrypt the Private Key for transport to the Subordinate CA